.npmrc

```
@@ -0,0 +1,5 @@
+audit=false
+fund=false
+@fortawesome:registry=https://npm.fontawesome.com/
+@awesome.me:registry=https://npm.fontawesome.com/
+//npm.fontawesome.com/:_authToken=${FONTAWESOME_PACKAGE_TOKEN}
```

```suggestion
//npm.fontawesome.com/:_authToken=${FONTAWESOME_PACKAGE_TOKEN}

```

* let's add new lie on the end